wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -27,8 +27,14 @@
                                 {{ message|safe }}
                             </li>
                         {% endfor %}
-                    </ul>
-                {% endif %}
+                    {% endif %}
+                </ul>
+                <template data-w-messages-target="templates">
+                    <li class="other" data-message-status="other">{% icon name="warning" classname="messages-icon" %}<span></span></li>
+                    <li class="error" data-message-status="error">{% icon name="warning" classname="messages-icon" %}<span></span></li>
+                    <li class="warning" data-message-status="warning">{% icon name="warning" classname="messages-icon" %}<span></span></li>
+                    <li class="success" data-message-status="success">{% icon name="success" classname="messages-icon" %}<span></span></li>
+                </template>
```

Comment:
```suggestion
                <template data-w-messages-target="templates">
                    <li class="success" data-message-status="success">{% icon name="success" classname="messages-icon" %}<span></span></li>
                    <li class="error" data-message-status="error">{% icon name="warning" classname="messages-icon" %}<span></span></li>
                    <li class="warning" data-message-status="warning">{% icon name="warning" classname="messages-icon" %}<span></span></li>
                </template>
```

sorry for the changing direction here, let's keep it simple with THREE options, but put success first and then in our controller if we cannot find any matching template child we will just grab the 'first' one.

This means that the 'success' message will be the default no matter how the data is provided, even if you provide something like `status: 'blah'`, it should try to find a suitable template element.